docs/user/firststeps.rst

```diff
@@ -156,34 +156,166 @@ Opening the file, we can see it contains some data in CSV format.
 Install and manage Python packages
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Continuing in the same terminal session as in last step, we can install python
-packages as usual with ``pip``:
+Here we will see how to use `conda <https://conda.io/docs/index.html>`_
```

Comment:
Maybe we should say conda is one way of managing environments and point to a doc somewhere explaining differences between virtual env and conda